server/src/sim.rs

```diff
@@ -478,6 +479,9 @@ impl Sim {
                 .voxel_data
                 .push((chunk_id, voxels.serialize(self.cfg.chunk_size)));
         }
+        for (&chunk_id, voxels) in self.preloaded_voxel_data.iter() {
+            (spawns.voxel_data).push((chunk_id, voxels.serialize(self.cfg.chunk_size)));
```

Comment:
Extraneous parens?

Reply:
Maybe. I mainly added them to get rustfmt not to unnecessarily splay things out onto multiple lines.

Reply:
I think having idiosyncratic syntax is worse than having a few extra linebreaks. If you want things more compact, better to tune the rustfmt config for the whole workspace, I think.

Reply:
Fair point. I'll go ahead and revert this. I wonder why there's no clippy lint for it.